torch/_inductor/codegen/wrapper.py

```diff
@@ -634,6 +634,13 @@ def generate_extern_kernel_alloc(self, extern_kernel, args):
             # view operation fallbacks cause issues since inductor
             # doesn't know the memory is still needed and might reuse it.
             ending = f".clone(){ending}"
+
+        # Note: 'from' is python keyword, it is not allowed to appear in python code
+        #       Although, only aten.random_ has an 'from' overload for now, we handle all possible cases
+        if kernel_name.endswith(".from"):
```

Comment:
Can we generalize this handling by using https://docs.python.org/3/library/keyword.html ?